text/0000-add-author.md

```diff
@@ -0,0 +1,55 @@
+# Meta
+[meta]: #meta
+- Name: Add Author to RFC Metadata
+- Start Date: 2020-03-14
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes: (put "N/A" unless this replaces an existing RFC, then link to that RFC)
+
+# Summary
+[summary]: #summary
+
+This is a proposal to add an `Author(s):` field to the `Meta` section of the RFC template.
+
+# Motivation
+[motivation]: #motivation
+
+When an RFC is merged, and then renamed, the commit history is lost. This makes it difficult to remember who wrote the RFC.
+
+# What it is
+[what-it-is]: #what-it-is
+
+A new `Author(s):` field in the Meta section of the RFC template.
+
+# How it Works
+[how-it-works]: #how-it-works
+
+You're not going to believe this, but you add an `Author(s):` field to the Meta section of the RFC template.
+
+# Drawbacks
+[drawbacks]: #drawbacks
+
+Even though the author will be know, the commit history is still hard to find.
+
+# Alternatives
+[alternatives]: #alternatives
+
+## RFC PR Link
+
+Instead of an author field, we could enforce that a link to the original RFC PR be added to the RFC after it's merged. This would make it easier to find the author and the history.
```

Comment:
We should be doing this already like [here](https://github.com/buildpacks/rfcs/blob/master/text/0003-pack-inspect-image.md).

Reply:
Maybe the name could be improved. `RFC PR` would disambiguate between the rfc PR itself and a draft PR implementing the RFC.

And while we are at it, `CNB Issue` appears blank in most merged RFCs. Maybe `Related Issues` instead, to optionally link to any relevant issues?